### ISSUE TITLE
Disagreeing option values for '--search_task': 'hook' vs 'sequence'

### ISSUE THREAD

**Issue by AlexanderRadionov** [JohnLangford/vowpal_wabbit#1297]
Python library test for search does not work properly.

- cd python/src/library
- make
- ./test_search
gets:

>num_actions = 4
output = [ 1 2 3 1 4 2 ]
should have printed: 1 2 3 1 4 2
num_actions = 4
output = [ 1 2 3 1 4 2 ]
should have printed: 1 2 3 1 4 2
final_regressor = sequence.model
Num weight bits = 18
learning rate = 0.5
initial_t = 0
power_t = 0.5
decay_learning_rate = 1
creating cache_file = sequence.data.cache
Reading datafile = sequence.data
can't open 'sequence.data', sailing on!
num sources = 0`

>finished run
number of examples per pass = 0
passes used = 20
weighted example sum = 0.000000
weighted label sum = 0.000000
average loss = n.a.
total feature number = 0
only testing
Error: Disagreeing option values for '--search_task': 'hook' vs 'sequence'`

>finished run
number of examples = 0
weighted example sum = 0.000000
weighted label sum = 0.000000
average loss = n.a.
total feature number = 0
terminate called after throwing an instance of 'VW::vw_argument_disagreement_exception'
  what():  Disagreeing option values for '--search_task': 'hook' vs 'sequence'

**Comment by JohnLangford** [comment]
Hal, can you take a look at this?

-John

On Tue, Jul 25, 2017 at 3:52 AM, AlexanderRadionov <notifications@github.com

**Comment by hal3** [comment]
Oh shoot I might have just broken this. Is this with my latest push or does it predate that?

**Comment by AlexanderRadionov** [comment]
@hal3 This broken previously. And your latest push does not fixed problem. I face the same error message.

**Comment by JohnLangford** [comment]
Hmm.   Hal, can you replicate?

-John

**Comment by hal3** [comment]
Ah sorry, I thought this was a Python thing. I know the bug. It'll be fixed soon :)

**Comment by AlexanderRadionov** [comment]
@hal3 Fixed. Thank you!
